pilot/pkg/model/service.go

```diff
@@ -354,6 +354,7 @@ const (
 // - consul: defaults to 'instance.Datacenter'
 //
 // This is used by EDS to group the endpoints by AZ and by .
+// TODO: remove me?
```

Comment:
this is used by EDS here: https://github.com/istio/istio/blob/release-1.0/pilot/pkg/proxy/envoy/v2/eds.go#L216

Reply:
not sure if we want to keep the EDS logic for something in the future

Reply:
Also keep.